tests/test_informationmodel.py

```diff
@@ -121,9 +124,14 @@ def test_to_graph_should_return_publisher() -> None:
     @prefix dcat: <http://www.w3.org/ns/dcat#> .
     @prefix foaf:  <http://xmlns.com/foaf/0.1/> .
 
-    <http://example.com/informationmodels/1>  a dcat:Resource ;
-    dct:publisher <https://example.com/organizations/1> ;
-    .
+    <http://example.com/informationmodels/1> a <http://www.w3.org/ns/dcat#Resource> .
```

Comment:
denne ser veldig snodig ut...

Punktum etter dcat:Resource er vel feil.
Og hvorfor oppgir du ressursen `<http://example.com/informationmodels/1>` to ganger? Fordi den forrige ble avsluttet for tidlig?

Ville regnet med noe slikt som dette:
```
<http://example.com/informationmodels/1>
    a    dcat:Resource ;
    dct:publisher    <https://example.com/organizations/1> ;
    dct:title    "CRD IV - Likviditet NSFR - konsolidert (KRT-1075)"@nb .
```

Reply:
Har sjekket syntaksen og den er visst riktig i følge turtle-validatoren :  http://ttl.summerofcode.be/

Men enig i det med serialiseringen, jeg skal se på det

Reply:
er ikke nøvendigvis noe feil med turtle-syntaks, men som menneskelesrbar turtle vil jeg si den feiler sterkt

Reply:
Litt usikker på hvordan jeg skal få til akkurat dette. Er jo også første gangen vi gjør det. RDF-messig så er det jo det samme, så det er jo egentlig bare snakk om litt syntaktisk sukker. Det er to forskjellige tripler, det ene sier at informationmodel er en en dcat:Resource, det andre sier at informationmodel har en dct:publisher som er https://example.com/organizations/1

Men de to forskjellige visningene er jo semantisk det samme, det er jo bare i visningen vår at det er litt forskjellig. Men hvis du vil, så kan jeg forske på hvordan man skal slå sammen disse tingene.

Reply:
Er ikke dette bare forventet resultat i en test, er vel bare å endre det...

Reply:
Etter å sjekket litt på Google så ser det ut som om den syntaksen du spørr etter er default når man bruker blank node. Så da må vi gjøre det i stedet for å få til dette. Men vi besluttet jo å gjøre dette med graf-union

Reply:
Det har vel ingenting med dette å gjøre.

Du lager en turtle som er kilde til expected i en test, så skriv turtle som er så menneskelesbar som mulig.

Reply:
Hei igjen @NilsOveTen . Jeg beklager, jeg tror jeg har misforstått deg. Jeg endrer nå testen til følgende: 

```<http://example.com/informationmodels/1>
    a    dcat:Resource ;
    dct:publisher    <https://example.com/organizations/1> ;
    dct:title    "CRD IV - Likviditet NSFR - konsolidert (KRT-1075)"@nb .
```
Det ene eller det andre er jo logisk ekvivalent for parseren. Jeg trodde at du etterspurte en eller annen form for funksjonell endring

Reply:
Er vel strengt tatt ingenting jeg har kommentert som har etterspurt funksjonell endring :P

Reply:
He, he nei jeg ser det nå. ;-) Problemet er bare hjernen min som virker dårligere når klokken begynner å nærme seg 15:00... :-P